packages/core/typescript/TypeScriptFileUpdate.ts

```diff
@@ -78,8 +78,10 @@ export class TypeScriptFileUpdate {
 		routesVariable = DEFAULT_ROUTES_VARIABLE,
 		lazyload = false,
 		routesPath = "",
-		root = false) {
-		this.addRouteModuleEntry(filePath, linkPath, linkText, routesVariable, parentRoutePath, lazyload, routesPath, root);
+		root = false,
```

Comment:
it will be great if parameters are positioned in the same way. Now we have four parameters on one row and all the rest each on its own row :(